internals/src/error/parse_error.rs

```diff
@@ -6,7 +6,7 @@
 /// [`InputString`](super::InputString), the type of `source` is specified as the second argument
 /// to the macro.
 ///
-/// The resulting type is public, conditionally implements [`std::error::Error`] and has a private
+/// The resulting type is public, conditionally implements `std::error::Error` and has a private
```

Comment:
I don't think doc has to be `no_std`.

Reply:
I can't seem to get the build error I was getting the other day, will remove for now.

Reply:
Ah found it, I was not enabling "std" when adding the `internals` dep.